### ISSUE TITLE
Installer crashes when using custom UIs with Features

### ISSUE THREAD

**Issue by benjveil** [oleg-shilo/wixsharp#1464]
Hi,

I created a small setup from the sample `wixsharp\Source\src\WixSharp.Samples\Wix# Samples\MergeModules\Consuming MergeModule\setup.cs`, and it works great out of the box.

When I switch the UI to ManagedUI and add the Features dialog, the setup crash when it gets to that dialog with this exception
in the log:

```
...

SFXCA: Binding to CLR version v4.0.30319
Managed Dialog unhandled Exception: WixToolset.Dtf.WindowsInstaller.BadQuerySyntaxException: SQL query syntax invalid or unsupported.
   at WixToolset.Dtf.WindowsInstaller.Database.OpenView(String sqlFormat, Object args)
   at WixSharp.Extensions.OpenView(Session session, String sqlText)
   at WixSharp.UI.Forms.FeatureItem..ctor(Session session, String name)
   at WixSharp.MsiSessionAdapter.<get_Features>b__8_1(String name)
   at System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   
   ...
```

Any ideas?

Here is the full code:

```cs
using WixSharp;
using WixSharp.Forms;

class Script
{
    static public void Main()
    {
        var featureA = new Feature("Feature A", "Feature A description");
        var featureB = new Feature("Feature B", "Feature B description");
        var complete = new Feature("Complete");

        complete.Add(featureA)
                .Add(featureB);

        var project =
                new ManagedProject("MyMergeModuleSetup", // Switched to ManagedProject
                    new MediaTemplate { CompressionLevel = CompressionLevel.none, EmbedCab = false },
                    new Dir(@"%ProgramFiles%\My Company",
                        new Merge(featureB, @"Files\MyMergeModule.msm"),
                        new Merge(featureB, @"Files\MyMergeModule1.msm")),
                    new EnvironmentVariable("foo", "bar"));

        project.Media.Clear();
        project.DefaultFeature = complete;
        
        
        // Changed this:
        //project.UI = WUI.WixUI_FeatureTree;

        // to this:
        project.ManagedUI = new ManagedUI();
        project.ManagedUI.InstallDialogs
                .Add(Dialogs.Welcome)
                .Add(Dialogs.Features)
                .Add(Dialogs.Progress)
                .Add(Dialogs.Exit);


        project.InstallerVersion = 200;

        project.BuildMsi();
    }
}
```

Thank you!

**Comment by oleg-shilo** [comment]
It's an unfortunate defect in the implementation of FeatureItem.
It's fixed now. I will release it asap

**Comment by oleg-shilo** [comment]
It's fixed in the latest v1.25.2.0 release. Please pull the latest nuget package with the fix.

**Comment by benjveil** [comment]
Wow, thanks a lot for the quick fix!  It works great!
